From 03fda8c5c35ab34850985960d918c28cc101304d Mon Sep 17 00:00:00 2001
From: Loc Vu <loc.vu.zn@renesas.com>
Date: Tue, 14 Jul 2020 09:55:59 +0700
Subject: [PATCH 021/433] ARM: dts: r8a7743: Add more clock source to R8A7743
 dtsi

This patch adds ssp_clk, ssprs_clk, i_clk, imp_clk nodes
to r8a7743 device tree.

Signed-off-by: Loc Vu <loc.vu.zn@renesas.com>
---
 arch/arm/boot/dts/r8a7743.dtsi | 32 ++++++++++++++++++++++++++++++++
 1 file changed, 32 insertions(+)

diff --git a/arch/arm/boot/dts/r8a7743.dtsi b/arch/arm/boot/dts/r8a7743.dtsi
index 8282f75..8c8bb62 100644
--- a/arch/arm/boot/dts/r8a7743.dtsi
+++ b/arch/arm/boot/dts/r8a7743.dtsi
@@ -1101,6 +1101,22 @@
 				#power-domain-cells = <0>;
 			};
 
+			ssp_clk: ssp_clk@e6150248 {
+				compatible = "renesas,r8a7743-div6-clock",
+					     "renesas,cpg-div6-clock";
+				reg = <0 0xe6150248 0 4>;
+				clocks = <&pll1_div2_clk>;
+				#clock-cells = <0>;
+				clock-output-names = "ssp";
+			};
+			ssprs_clk: ssprs_clk@e615024c {
+				compatible = "renesas,r8a7743-div6-clock",
+					     "renesas,cpg-div6-clock";
+				reg = <0 0xe615024c 0 4>;
+				clocks = <&pll1_div2_clk>;
+				#clock-cells = <0>;
+				clock-output-names = "ssprs";
+			};
 			/* Variable factor clocks */
 			sd2_clk: sd2_clk@e6150078 {
 				compatible = "renesas,r8a7743-div6-clock",
@@ -1168,6 +1184,14 @@
 				clock-mult = <1>;
 				clock-output-names = "hp";
 			};
+			i_clk: i_clk {
+				compatible = "fixed-factor-clock";
+				clocks = <&cpg_clocks R8A7743_CLK_PLL1>;
+				#clock-cells = <0>;
+				clock-div = <2>;
+				clock-mult = <1>;
+				clock-output-names = "i";
+			};
 			b_clk: b_clk {
 				compatible = "fixed-factor-clock";
 				clocks = <&cpg_clocks R8A7743_CLK_PLL1>;
@@ -1200,6 +1224,14 @@
 				clock-mult = <1>;
 				clock-output-names = "m2";
 			};
+			imp_clk: imp_clk {
+				compatible = "fixed-factor-clock";
+				clocks = <&cpg_clocks R8A7743_CLK_PLL1>;
+				#clock-cells = <0>;
+				clock-div = <4>;
+				clock-mult = <1>;
+				clock-output-names = "imp";
+			};
 			rclk_clk: rclk_clk {
 				compatible = "fixed-factor-clock";
 				clocks = <&cpg_clocks R8A7743_CLK_PLL1>;
-- 
2.7.4

